agent/os_helper.py

```diff
@@ -380,3 +380,25 @@ def reboot_required():
                                        key=cmp_to_key(rpm.versionCompare), reverse=True)[0]
             return rpm.versionCompare(latest_kernel_pkg, kernel_pkg) > 0
     return None
+
+
+def upgrade_packages(pkg_names):
```

Comment:
Some thoughts:
* We need to log/print the packages that will be upgrade
* We need to ask for confirmation from the user before we proceed

Reply:
done